opm/simulators/flow/AluGridVanguard.hpp

```diff
@@ -232,6 +235,13 @@ class AluGridVanguard : public FlowBaseVanguard<TypeTag>
     const CartesianIndexMapper& cartesianIndexMapper() const
     { return *cartesianIndexMapper_; }
 
+    /*!
+     * \brief Returns the object which maps a global element index of the simulation grid
+     *        to the corresponding element index of the level logically Cartesian index.
+     */
+    const LevelCartesianIndexMapper levelCartesianIndexMapper() const // Consider adding data member to reduce/optimize memory-use?
```

Comment:
Resolve the comment please.

Reply:
Comment has been removed. Decision will be taken in the future with a clearer idea of how many times it needs to be called.